qt/widgets/common/src/NonOrthogonal.cpp

```diff
@@ -115,8 +115,15 @@ void doProvideSkewMatrix(Mantid::Kernel::DblMatrix &skewMatrix,
   Mantid::Kernel::Matrix<Mantid::coord_t> affineMatrix;
   try {
     auto const *transform = workspace.getTransformToOriginal();
-    affineMatrix = transform->makeAffineMatrix();
-  } catch (std::runtime_error &) {
+    if (transform)
+      affineMatrix = transform->makeAffineMatrix();
+    else {
+      // Create identity matrix of dimension+1
+      std::size_t nDims = workspace.getNumDims() + 1;
+      affineMatrix =
+          Mantid::Kernel::Matrix<Mantid::coord_t>(nDims, nDims, true);
+    }
+  } catch (...) {
```

Comment:
New behavior, catching all errors?

Reply:
Not sure why I did this.  Changed back to original.

Reply:
changed to old error catching